BuyPolish/Pola/UI/ProductSearch/About/BPAboutViewControllerCell.m

```diff
@@ -0,0 +1,41 @@
+#import "BPAboutViewControllerCell.h"
+#import "BPTheme.h"
+
+CGFloat const WHITE_BACKGROUND_HORIZONTAL_MARGIN = 8.0f;
+CGFloat const WHITE_BACKGROUND_VERTICAL_MARGIN = 16.0f;
+CGFloat const LABEL_X_OFFSET = 20.0f;
+
+@interface BPAboutViewControllerCell ()
+@property (nonatomic) UIView *whiteBackgroundView;
+@end
+
+@implementation BPAboutViewControllerCell
+
+- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
```

Comment:
There should be no option of creating cell with different style than `UITableViewCellStyleDefault`, so trim constructor to `initWithReuseIdentifier:` and make `[super initWithStyle:UITableViewCellStyleDefault reuseIdentifier:reuseIdentifier]` 2 lines below